.circleci/config.yml

```diff
@@ -157,6 +157,104 @@ references:
         python3 --version
         demisto-sdk --version
 
+  download_configuration: &download_configuration
+    run:
+      name: Download Configuration
+      when: always
+      command: |
+        if ./Tests/scripts/is_ami.sh ;
+          then
+            ./Tests/scripts/download_demisto_conf.sh
+
+          else
+            ./Tests/lastest_server_build_scripts/download_demisto_conf.sh
+        fi
+  
+  test_download_upload: &test_download_upload
```

Comment:
This step is used only once, why adding it to references and not the job iteself?

Reply:
To make the steps much more readable

Reply:
Then make it more readable by separating it to different steps then.

In this single step you're:
1. Uploading.
2. Downloading.
3. Formatting
4. Validating
5. Linting

Each can be a step on its own, which will make the whole process clearer.

And then you could also remove the redundant exit code parsing